src/tree/browser/components/utils.tsx

```diff
@@ -0,0 +1,181 @@
+/********************************************************************************
+ * Copyright (C) 2024 EclipseSource and others.
```

Comment:
Would it make sense to split into TS and React helpers? No hard feelings about this, just felt like a clearer separation.